src/group/orbit/factored_transversal.rs

```diff
@@ -14,6 +14,30 @@ pub struct FactoredTransversal {
     pub(super) transversal: HashMap<usize, Permutation>,
 }
 
+#[allow(clippy::implicit_hasher)]
```

Comment:
Just for completeness, we can fix this lint by adding a type parameter <S: std::hash::BuildHasher>.
and changing transversal: &HashMap<usize, Permutation, S>.

Reply:
Done now